molecule/packages-static/group_vars/all

```diff
@@ -4,6 +4,7 @@ pulp_upgrade: true
 pulp_install_source: packages
 pulp_api_bind: "unix:/var/run/pulpcore-api/pulpcore-api.sock"
 pulp_content_bind: "unix:/var/run/pulpcore-content/pulpcore-content.sock"
+pulp_redis_bind: "unix:/var/run/redis/redis.sock"
```

Comment:
👍🏽

Reply:
Yup! This will actually test the functionality.